tcn_hpl/data/vectorize/locs_and_confs.py

```diff
@@ -0,0 +1,141 @@
+import functools
+import typing as tg
+
+import numpy as np
+from numpy import typing as npt
+
+from tcn_hpl.data.vectorize._interface import Vectorize, FrameData
+from tcn_hpl.data.vectorize_classic import (
+    obj_det2d_set_to_feature,
+    zero_joint_offset,
+    HAND_STR_LEFT,
+    HAND_STR_RIGHT,
+)
+
+
+class LocsAndConfs(Vectorize):
+    """
+    Previous manual approach to vectorization.
+
+    Arguments:
+        feat_version: Version number of the feature to produce.
+        top_k: The number of top per-class examples to use in vector
+            construction.
```

Comment:
Please update this doc-string to have an appropriate description, and update arguments to reflect the constructor.

Reply:
Done. Thanks.